src/index.html

```diff
@@ -5,8 +5,420 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="menu">
+      <a href="#main" class="logo">
+        <img src="images/logo.png" alt="LOGO">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
```

Comment:
wrap if you have more than 2 params or length > 80
```suggestion
            <a href="#apple" class="nav__link is-active">
```